apps/masterbots.ai/app/p/[username]/page.tsx

```
@@ -0,0 +1,10 @@
+import ChatLayout from "./layout";
+export default async function ProfilePage({ params }: any) {
```

Biome

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)